src/services/decoratemobiledrawinteraction.js

```diff
@@ -0,0 +1,93 @@
+goog.provide('ngeo.DecorateMobileDrawInteraction');
```

Comment:
I would not do a specific decorator for this.
ngeo decorators are here to make generic ol3 components interfacable with angular. `decorateMobileDrawInteraction` is specific for a specific interaction `ngeo.interaction.MobileDraw`. I don't know what you think about this, but i would put the define properties inside the interaction itself, during creation.